ack/readers/awin_advertiser/config.py

```diff
@@ -0,0 +1,55 @@
+# GNU Lesser General Public License v3.0 only
+# Copyright (C) 2020 Artefact
+# licence-information@artefact.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+from datetime import datetime
+from typing import Tuple
+
+from pydantic import BaseModel, validator
+
+DATEFORMAT = "%Y-%m-%d"
+
+AWIN_API_ENDPOINT = "https://api.awin.com/advertisers/"
+LIMIT_NVIEWS_PER_REQ = 5
+MAX_WAIT_REPORT_DELAY = 4096
+
+# https://wiki.awin.com/index.php/Advertiser_Service_API
+REPORT_TYPES = [
+    "publisher",
+    "creative",
+    "campaign"
+]
+
+
+class AwinAdvertiserReaderConfig(BaseModel):
+    auth_token: str
+    advertiser_id: str
+    report_type: str
+    region: str
+    timezone: str
+    start_date: datetime = None
+    end_date: datetime = None
+    date_range: Tuple[datetime, datetime] = None
+    # day_range: Literal[DAY_RANGE] = None
```

Comment:
If you want to implement a data range, it's already done in data handler I think to have the last 7 days, last month or else.

Reply:
This is not needed, so it has been removed